setup.py

```diff
@@ -5,7 +5,7 @@
 from setuptools.command.test import test as TestCommand
 import glob
 
-__version__ = '1.7.3'
+__version__ = '1.7.4'
```

Comment:
If you're dropping 2.7 and 3.4, the classifiers=[] at the bottom of setup.py needs to have a few things deleted.